dist.ini

```diff
@@ -31,7 +31,6 @@ perl = 5.014000
 [NextRelease]
 [VersionFromModule]
 [ReversionOnRelease]
-[PkgVersion]
```

Comment:
Why remove this line ?

Reply:
this should duplicate with VersionFromModule

Reply:
I remember they are different plugins. `VersionFromModule` will extract version number from module and build the package like mypackage-1.0.0.tar.gz .And `PkgVersion` will add version information to the package.

At last we changed to `OurPkgVersion` in our template https://github.com/regentmarkets/perl-Dist-Zilla-MintingProfile-BINARY/blob/master/profiles/private/skel/dist.ini#L33